MDANSE/Src/MDANSE/Framework/Jobs/Eccentricity.py

```diff
@@ -22,7 +22,8 @@
 
 
 class Eccentricity(IJob):
-    """Computes the eccentricity for a set of atoms e.g. in a micelle.
+    """Computes the eccentricity of a selected set of atoms.
+
     The eccentricity is calculated from the principal moments of
     inertia via the equation sqrt(pm3**2 - pm1**2) / pm3 where pm1 and
     pm3 are the smallest and largest principal moments of inertia
```

Comment:
```suggestion
    The eccentricity is calculated from the principal moments of
    inertia via the equation :math:`\sqrt(\text{pm3}^{2} - \text{pm1}^{2}) / \text{pm3}` where :math:`\text{pm1}` and
    :math:`\text{pm3}` are the smallest and largest principal moments of inertia
```

Updating maths to use math font.